src/main/java/com/softdevsix/domain/repositories/IProjectRepository.java

```diff
@@ -9,9 +10,10 @@
 
 @Repository
 public interface IProjectRepository {
```

Comment:
The repository should extend from `JpaRepository` and its generic types, so we won't have to implement some methods from zero.